modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/DefaultExecutionMediator.java

```diff
@@ -146,7 +150,11 @@ private OperationExecutionResult executeWithInterceptors(OperationExecutor execu
       after(context, result, interceptors);
```

Comment:
Is it safe to assume all before/after interceptors are ok to be run on an event loop?